### PR TITLE
August Ratification Polls

### PR DESCRIPTION
**Pull request by blimpa**
- DAIF-001 budget
- Core Unit Offboarding Process Amendment

### CODE REVIEW

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
Didn't do a full review - just spotted a couple of things when I glanced at it

File: governance/polls/Ratification Poll - Core Unit Offboarding Process Amendments (MIP4c2-SP15) - August 8, 2022.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, August 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+  The community may vote in this poll to express support or opposition to MIP4c2-SP15 being accepted and implemented in the Maker Protocol.
```
Comment (patrick-j-govalpha) [inline governance/polls/Ratification Poll - Core Unit Offboarding Process Amendments (MIP4c2-SP15) - August 8, 2022.md:29]: Can you fix the weird indent here?

File: governance/polls/Ratification Poll - Modify Dai Foundation Core Unit Budget (DAIF-001) - August 8, 2022.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: Ratification Poll for Modify Dai Foundation Core Unit Budget (MIP40c3-SP74) - August 8, 2022 
+summary: This Budget Proposal for DAI expenses aims to fund the activities of the DAI Foundation Core Unit (DAIF-001) as described in MIP39c2-SP17 adds Core Unit DAIF-001: Dai Foundation.
```
Comment (patrick-j-govalpha) [inline governance/polls/Ratification Poll - Modify Dai Foundation Core Unit Budget (DAIF-001) - August 8, 2022.md:3]: Summary seems wrong, this isn't adding DAIF-001 and also can't have a colon in the yaml

File: governance/polls/Ratification Poll - Modify Dai Foundation Core Unit Budget (DAIF-001) - August 8, 2022.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp74-modify-dai-foundation-core-unit-budget-daif-001/16379/5
```
Comment (patrick-j-govalpha) [inline governance/polls/Ratification Poll - Modify Dai Foundation Core Unit Budget (DAIF-001) - August 8, 2022.md:4]: Link to the proposal, not to the summary comment

File: governance/polls/Ratification Poll - Modify Oracle Core Unit Budget (ORA-001) - August 8, 2022.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+* **Option 1:** Approve existing budget (2021-2022): **5,061,125 DAI** (Business as usual).
+* **Option 2:** Approve proposed budget (2022-2023): **4,554,575 DAI** (Reduced budget exempt from further cuts for 12 months).
+* **Option 3:** Reject budget.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*"MIP40c3-SP75 renews the Oracle Core Unit ORA-001 budget from July 1, 2022 through June 30, 2023."*
+
+This budget also includes an [MKR compensation schedule](https://github.com/makerdao/mips/blob/317edeae2da5667876f21bdf4d23c04222754819/MIP40/MIP40c3-Subproposals/MIP40c3-SP75.md#mkr-vesting).
```
Comment (patrick-j-govalpha) [inline governance/polls/Ratification Poll - Modify Oracle Core Unit Budget (ORA-001) - August 8, 2022.md:40]:
```suggestion
This budget also includes an updated [MKR compensation schedule](https://github.com/makerdao/mips/blob/317edeae2da5667876f21bdf4d23c04222754819/MIP40/MIP40c3-Subproposals/MIP40c3-SP75.md#mkr-vesting) based on the previously approved MKR budget proposal for this Core Unit.
```

File: governance/polls/Ratification Poll - Modify Oracle Core Unit Budget (ORA-001) - August 8, 2022.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    victory_conditions:
+        - { type : instant-runoff }
+    result_display: instant-runoff-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Approve existing budget (2021-2022) (Business as usual)
+   2: Approve proposed budget (2022-2023) (Reduced budget exempt from further cuts for 12 months)
+   3: Reject Budget
+start_date:2022-08-08T16:00:00
```
Comment (patrick-j-govalpha) [inline governance/polls/Ratification Poll - Modify Oracle Core Unit Budget (ORA-001) - August 8, 2022.md:16]: add space - I think this is breaking the formatting - can be seen on rich diff view
```suggestion
start_date: 2022-08-08T16:00:00
```

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
see comments

File: governance/polls/Ratification Poll - Modify Dai Foundation Core Unit Budget (DAIF-001) - August 8, 2022.md (hunk 2 above)
Comment (blimpa) [reply governance/polls/Ratification Poll - Modify Dai Foundation Core Unit Budget (DAIF-001) - August 8, 2022.md:3]: image

That's the Sentence Summary. I'm modifying it a bit for more clarity

**Review by patrick-j-govalpha — APPROVED** [review]
LGTM

**Pull request merged**